integration-tests/deployment/ccip/test_assertions.go

```diff
@@ -72,7 +72,9 @@ func ConfirmCommitWithExpectedSeqNumRange(
 	}
 
 	defer subscription.Unsubscribe()
-	timer := time.NewTimer(5 * time.Minute)
+	// sometimes the commit is delayed for finality, hence setting timeout to 20min for live chains
+	// TODO - make this configurable
+	timer := time.NewTimer(20 * time.Minute)
```

Comment:
This should not be done here, the test deadline should be appropriately set IMO.

Reply:
I don't think that's much better, maybe just use the test deadline to inform the timer?

```golang
var duration time.Duration
	deadline, ok := t.Deadline()
	if ok {
		duration = deadline.Sub(time.Now())
	} else {
		duration = 5 * time.Minute
	}
timer := time.NewTimer(duration)
```

Or maybe just don't even have this timer, and when the test times out, it times out? I don't think we had this timer in the initial verion of the integration test, but not 100% sure